gradle/libs.versions.toml

```diff
@@ -1,95 +1,98 @@
 [versions]
-bouncyCastle = "1.70"
+asset-model = "0.1.2"
+bouncycastle = "1.70"
 exposed = "0.37.3"
+figure-eventstream = "0.7.1"
 flyway = "8.0.2"
 grpc = "1.45.1" # If updating this value, ensure to update the doc reference in the ClientConfig class
-grpcSpringboot = "3.4.3"
-hikariCP = "5.0.1"
+grpc-springboot = "3.4.3"
+hikari = "5.0.1"
 jackson = "2.12.5"
-jacksonProtobuf = "0.9.12"
+jackson-protobuf = "0.9.12"
 jwt = "3.19.2"
 kotlin = "1.6.10"
-kotlinCoroutines = "1.5.2"
-kotlinLogging = "2.1.21"
+kotlin-coroutines = "1.5.2"
+kotlin-logging = "2.1.21"
 mockk = "1.12.3"
-okHttp = "4.9.1"
+okhttp = "4.9.1"
 postgres = "42.3.3"
 protobuf = "3.19.1"
-provenanceClient = "1.0.5"
-provenanceEventStream = "0.6.3"
-provenanceHdWallet = "0.1.15"
-provenanceProto = "1.8.0-rc10"
-provenanceScope = "0.4.9"
+provenance-client = "1.3.0"
+provenance-hdwallet = "0.1.15"
+provenance-proto = "1.13.0"
+provenance-scope = "0.4.9"
 scarlet = "0.1.12"
-sqLite = "3.36.0.3"
-testContainers = "1.17.3"
+sqlite = "3.36.0.3"
+testcontainers = "1.17.3"
 
 [libraries]
-bcProv = { module = "org.bouncycastle:bcprov-jdk15on", version.ref = "bouncyCastle" }
-grpcProtobuf = { module = "io.grpc:grpc-protobuf", version.ref = "grpc" }#, exclude = listOf("com.google.protobuf:protobuf-java", "com.google.protobuf:protobuf-java-util")).include()
-grpcStub = { module = "io.grpc:grpc-stub", version.ref = "grpc" }
-grpcSpringbootStarter = { module = "io.github.lognet:grpc-spring-boot-starter", version.ref = "grpcSpringboot" }
-grpcNetty = { module = "io.grpc:grpc-netty-shaded", version.ref = "grpc" }
-kotlinModule = { module = "com.fasterxml.jackson.module:jackson-module-kotlin", version.ref = "jackson" }
-protobufModule = { module = "com.hubspot.jackson:jackson-datatype-protobuf", version.ref = "jacksonProtobuf" }
-allOpen = { module = "org.jetbrains.kotlin:kotlin-allopen", version.ref = "kotlin" }
-coroutinesCoreJvm = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm", version.ref = "kotlinCoroutines" }
-coroutinesReactor = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-reactor", version.ref = "kotlinCoroutines" }
-coroutinesJdk8 = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-jdk8", version.ref = "kotlinCoroutines" }
-coroutinesSLF4J = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-slf4j", version.ref = "kotlinCoroutines" }
+# Core
+asset-model = { module = "io.provenance.model:metadata-asset-model", version.ref = "asset-model" }
```

Comment:
Moved this from a direct ref in a build file to this file, just for consolidation's sake